sdcm/cluster.py

```diff
@@ -3748,7 +3748,8 @@ def get_load_balancing_policy(self, whitelist_nodes: Optional[list[BaseNode]] =
             loader = self.nodes[0].test_config.tester_obj().loaders.nodes[0]
             loader_rack = loader.node_rack
             loader_dc = loader.datacenter
-            datacenter_name_per_region = self.get_datacenter_name_per_region(db_nodes=self.nodes)
+            nodes = whitelist_nodes or self.nodes  # need to get datacenter names from alive nodes if whitelist_nodes passed
```

Comment:
Does your test runs with `self.params.get("rack_aware_loader") and self.racks_count > 1`?
It appears that this code has no impact on your test. 
Since the rackaware test isn’t expected to trigger this type of nemesis, I believe this change is unnecessary.

Reply:
So this should be fixed - set `rack_aware_loader` to false in not relevant for rackaware test

Reply:
Without fix any test could be failed when appropriate conditions happened. It should don't affect current tests, because whitelist should be passed explicitly, and this done for now only in 2 nemesis.

Reply:
@fruch , fixed `get_datacenter_name_per_region` according to your comment.